src/scripts/generic/directives/fileread.js

```diff
@@ -0,0 +1,14 @@
+module.exports = /* @ngInject */function () {
+  return {
+    scope: {
+      fileread: '='
```

Comment:
don't like the name, generally this should be handled by `<input type="file" ng-model="$ctrl.input" />` but since it is not, we need this. Better name would be something like `file-model`